src/configs/shared.ts

```diff
@@ -25,7 +25,6 @@ export const DEFAULT_DISQUALIFY_TIME = "7 days"; // 7 days
 export const DEFAULT_CHAIN_ID = 1; // ethereum
 export const DEFAULT_RPC_ENDPOINT = "https://rpc-bot.ubq.fi/v1/mainnet";
 
-export const DEFAULT_PAYMENT_TOKEN = "0x6B175474E89094C44Da98b954EedeAC495271d0F";
```

Comment:
Shouldn't we replace this with another default instead of removing?

Reply:
We don't need the `DEFAULT_PAYMENT_TOKEN` variable anymore so it was deleted. Now we have a separate payment token for [each chain](https://github.com/ubiquity/ubiquibot/pull/316/files#diff-7a573dc385193e6a51c6e9a66e89d361cb981e4c10d27e77d398aa669d9f980dR22).